ui/apps/platform/webpack.ocp-plugin.config.js

```diff
@@ -0,0 +1,142 @@
+            sagas: path.resolve(__dirname, 'src/sagas'),
+            messages: path.resolve(__dirname, 'src/messages'),
+            mockData: path.resolve(__dirname, 'src/mockData'),
+            sorters: path.resolve(__dirname, 'src/sorters'),
+            'test-utils': path.resolve(__dirname, 'src/test-utils'),
+            images: path.resolve(__dirname, 'src/images'),
+            css: path.resolve(__dirname, 'src/css'),
+        },
+    },
+    module: {
+        rules: [
+            {
+                test: /\.(jsx?|tsx?)$/,
+                exclude: /\/node_modules\//,
+                use: [
+                    {
+                        loader: 'ts-loader',
+                        options: {
+                            transpileOnly: true,
+                            configFile: path.resolve(__dirname, 'tsconfig.json'),
+                        },
+                    },
+                ],
+            },
+            {
+                test: /\.(css)$/,
+                use: ['style-loader', 'css-loader'],
+            },
+            {
+                test: /\.(png|jpg|jpeg|gif|svg|woff2?|ttf|eot|otf)(\?.*$|$)/,
+                type: 'asset/resource',
+                generator: {
+                    filename: isProd ? 'assets/[contenthash][ext]' : 'assets/[name][ext]',
+                },
+            },
+            {
+                test: /\.(m?js)$/,
+                resolve: {
+                    fullySpecified: false,
+                },
+            },
+        ],
+    },
+    devServer: {
+        port: 9001,
+        // Allow Bridge running in a container to connect to the plugin dev server.
+        allowedHosts: 'all',
+        headers: {
+            'Access-Control-Allow-Origin': '*',
+            'Access-Control-Allow-Methods': 'GET, POST, PUT, DELETE, PATCH, OPTIONS',
+            'Access-Control-Allow-Headers': 'X-Requested-With, Content-Type, Authorization',
+        },
+        devMiddleware: {
+            // The ConsoleRemotePlugin sets a publicPath of '/api/plugins/<plugin-name>/', however when running the
+            // console locally in development mode, the proxy strips off this prefix and only leaves '/', which causes
+            // the plugin to not be able to find its assets.
+            publicPath: '/',
+        },
+    },
+    plugins: [
+        new ConsoleRemotePlugin({
+            validateSharedModules: false,
+            pluginMetadata: {
+                name: 'advanced-cluster-security',
+                version: '0.0.1',
+                displayName: 'Red Hat Advanced Cluster Security for OpenShift',
+                description: 'OCP Console Plugin for Advanced Cluster Security',
+                exposedModules: {
+                    SecurityVulnerabilitiesPage:
+                        './ConsolePlugin/SecurityVulnerabilitiesPage/Index',
+                },
+                dependencies: {
+                    '@console/pluginAPI': '>=4.19.0',
```

Comment:
Supports OCP >= 4.19.0.

We may be able to reduce this, but for ease of testing we are only worried about latest.